src/app/content/routes.ts

```diff
@@ -12,7 +12,12 @@ interface State {
 }
 
 export const content: Route<Params, State> = {
-  component: components.Content,
+  component: Loadable({
+    loader: () => import(/* webpackChunkName: "Content" */ './components/Content'),
+    loading: () => null,
```

Comment:
Should an indicator (visual & aural) be added so folks (screenreader, selenium, slow-internet users) know that something is happening?

And can it error when the chunk cannot be loaded?

Reply:
hmmmm. we should test something using aria-live to announce the page load

Reply:
I _think_ that since the JS is loaded immediately aria-live is not necessary but I'll spin up NVDA and send it to a page & see if the whole page is read out

Reply:
NVDA reads it fine